### PR TITLE
Add docs

### PR DESCRIPTION
**Pull request by 1ethanhansen**

### CODE REVIEW

**Comment by 1ethanhansen** [comment]
@kytrinyx it suggested you as a reviewer, if it should be someone else please let me know! thank you :)

File: docs/TESTS.md [hunk 1]
```diff
@@ -1,15 +1,11 @@
 # Tests
 
-<!-- TODO: write document
+## Running tests
+To run the tests run the command
 
-  This document should describe everything related to running tests in the track.
+```bash
+./bin/test
+```
```
Comment (ErikSchierboom) [inline docs/TESTS.md:8]: The intended audience for this file is a student, and it should describe how the student can run an exercise's tests. 
Looking at an exercise, I don't see a `bin/test` file in it.
However, this repo _does_ have a `bin/test` file, so you might have written this describing how to test the track?
If so, could you rewrite this to describe how the student should run the tests?

**Review by ErikSchierboom — COMMENTED** [review]
One small nit.

File: docs/TESTS.md (hunk 1 above)
Comment (1ethanhansen) [reply docs/TESTS.md:8]: Ah yes I did describe how to test the track rather than individual exercises. Will update, thanks!

Comment (1ethanhansen) [reply docs/TESTS.md]: Fixed in 88476d02dc1b5ddde5161265cc293f279bbf7e79

**Review by ErikSchierboom — APPROVED** [review]
Nice work!

**Pull request merged**